third_party/3/lxml/etree.pyi

```diff
@@ -21,14 +21,25 @@ class ElementChildIterator(Iterator['_Element']):
     def __iter__(self) -> 'ElementChildIterator': ...
     def __next__(self) -> '_Element': ...
 
+
+
 class _Element(Iterable['_Element']):
     def addprevious(self, element: '_Element') -> None: ...
-
+    def addnext(self, element: '_Element') -> None: ...
+    def append(self, element: '_Element') -> None: ...
+    def clear(self) -> None: ...
+    def get(self, key:_AnyStr, default: _AnyStr=None) -> _AnyStr: ...
```

Comment:
Whenever ``None`` is an acceptable argument, there should be an explicit ``Optional`` annotation.